.github/scripts/check-tag-format.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+# Checking if current tag matches the required formating
+is_pre_release=$1
+current_tag=$(echo $GITHUB_REF | cut -d '/' -f 3 | tr -d ' ',v)
+
+if [ $is_pre_release = false ]; then
+  # Works with the format vX.X.X
+  #
+  # Example of correct format:
+  # v0.1.0
+  echo "$current_tag" | grep -E "[0-9]*\.[0-9]*\.[0-9]*$"
+  if [ $? != 0 ]; then
```

Comment:
A function would be nice to avoid repeating the code

Reply:
A function that takes the regex as a parameter?

Reply:
Sorry, wrong line, with the `if [ $? != 0 ]; then` until `echo 'Please refer to the contributing guide for help.'
    exit 1`